tensorflow/python/keras/engine/topology_test.py

```diff
@@ -42,6 +42,34 @@
 
 class TopologyConstructionTest(test.TestCase):
 
+  def test_add_variable_supports_TensorShape(self):
```

Comment:
This test appears unrelated to the CL; the CL adds support for TensorShape in initializers, and support in add_variable is just a side-effect. Please just add a unit test in the initializers tests.

Reply:
Could you explain what's CL? changed line? And could you clarity what the new unit test is for? I think init_op_test has added tests for TensorShape.